packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -252,3 +252,39 @@ class ByType extends SerializableFinder {
     return new ByType(json['type']);
   }
 }
+
+
+/// A Flutter driver command that retrieves a semantics id using a specified
+/// finder.
+///
+/// Semantics must be enabled before using this command.
```

Comment:
Link to how to enable semantics?

Reply:
Addressed in https://github.com/flutter/flutter/pull/19047